docs/usage/tables/parquet.md

```diff
@@ -0,0 +1,120 @@
+├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│STORAGE_LOCATION    │file:///Users/avriiil/Documents/git/my-forks/unitycatalog/etc/data/external/unity/default/tables/te│
+│                    │st                                                                                                  │
+├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│COMMENT             │null                                                                                                │
+├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│PROPERTIES          │{}                                                                                                  │
+├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│CREATED_AT          │1721727161471                                                                                       │
+├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│UPDATED_AT          │1721727161471                                                                                       │
+├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│TABLE_ID            │e95d4f58-6e6e-4c4c-b45f-d47befb07cde                                                                │
+└────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────┘
+```
+
+## Pros and Cons of Using Parquet
+
+[Apache Parquet](https://parquet.apache.org/) is a popular file format for tabular data. It is an efficient data storage format and offers great features like predicate pushdown, column pruning and partition skipping.
+
+Parquet also has some important drawbacks you should be aware of compared to open table formats like Delta Lake and Apache Iceberg.
+
+Here are some of the challenges of working with Parquet tables:
+
+- No ACID transactions for Parquet data lakes means it's easier to accidentally corrupt your data
+- It is not easy to delete rows from Parquet tables
+- No DML transactions
+- There is no change data feed
+- Slow file listing overhead
+- Expensive footer reads to gather statistics for file skipping
+- There is no way to rename, reorder, or drop columns without rewriting the whole table
+
+Open table formats like Apache Iceberg and Delta Lake were specifically designed to overcome these limitations. Storing your data in one of those formats is almost always more advantageous than storing it in Parquet files.
+
+You can read a detailed comparison of [Delta Lake vs Parquet](https://delta.io/blog/delta-lake-vs-parquet-comparison/) for more information.
```

Comment:
Suggest removing Delta specific language here.  Once we get the UC docs updated for Delta and Iceberg, it may make sense to point to the UC docs for these lakehosue formats.